src/plugins/walletConnectToDapps/v1/WalletConnectBridgeProvider.tsx

```diff
@@ -219,6 +249,39 @@ export const WalletConnectBridgeProvider: FC<PropsWithChildren> = ({ children })
     [handleDisconnect],
   )
 
+  const handleSwitchChain = useCallback(
+    (err: Error | null, payload: any) => {
+      if (err) return moduleLogger.error(err, 'handleSwitchChain')
+      if (!wcAccountId) return moduleLogger.error('No account id found for wallet connect')
+      moduleLogger.info('handleSwitchChain', payload)
+      const chainIdHex = payload.params[0].chainId
```

Comment:
we should make this slightly clearer so downstream code is less confusing for us later (e.g on this line `chainId: chainIdHex, // chain reference as hex`)

```
      const walletConnectChainIdHex = payload.params[0].chainId
```

or 

```
      const chainReferenceHex = payload.params[0].chainId
```

Reply:
https://github.com/shapeshift/web/pull/4590/commits/678b074201432aae9fb2db6a10d64424a8e0aaab